test/datalog_tests/simple3.dl

```diff
@@ -214,3 +214,79 @@ typedef Thosts = Thosts{id:Option<string>, capacity:Option<signed<64>>, up:Optio
 input relation Rhosts[Thosts] primary key (row) (row.id)
 output relation Rhostsv[Thosts]
 Rhostsv[v0] :- Rhosts[v],var v0 = v.
+
+
+/**********************************************************/
+
+// Test code generation for `match` expressions.
```

Comment:
no ast files for this?

Reply:
we don't have those anymore, they were a paint to maintain.